### ISSUE TITLE
Attendee doesn't republish local stream or doesn't reconnect at all after internet connection loss

### ISSUE THREAD

**Issue by andreisergiu98** [aws/amazon-chime-sdk-js#2560]
### What happened and what did you expect to happen?

When an attendee loses its internet connection, but then recovers it, chime throws the following error:

```
index.js:18 Uncaught TypeError: Cannot read properties of null (reading 'length')
    at Object.utf8_length [as length] (index.js:18:1)
    at Writer.write_string [as string] (writer.js:392:1)
    at Function.encode (SignalingProtocol.js:12295:1)
    at Function.encode (SignalingProtocol.js:12014:1)
    at Function.encode (SignalingProtocol.js:302:1)
    at DefaultSignalingClient.sendMessage (DefaultSignalingClient.ts:311:1)
    at DefaultSignalingClient.remoteVideoUpdate (DefaultSignalingClient.ts:192:1)
    at DefaultAudioVideoController.updateRemoteVideosFromLastVideosToReceive (DefaultAudioVideoController.ts:948:1)
    at DefaultAudioVideoController.update (DefaultAudioVideoController.ts:831:1)
    at ReceiveVideoStreamIndexTask.resubscribe (ReceiveVideoStreamIndexTask.ts:188:1)
    at ReceiveVideoStreamIndexTask.handleIndexFrame (ReceiveVideoStreamIndexTask.ts:114:1)
    at ReceiveVideoStreamIndexTask.handleSignalingClientEvent (ReceiveVideoStreamIndexTask.ts:62:1)
    at DefaultSignalingClient.sendEvent (DefaultSignalingClient.ts:404:1)
    at DefaultSignalingClient.receiveMessage (DefaultSignalingClient.ts:342:1)
    at WebSocket.<anonymous> (DefaultSignalingClient.ts:418:1)
```

If said attendee has the camera on, it won't be published to the other attendees so they'll see him with camera off. Sometimes the attendee won't reconnect at all, with the same error as above.

This happens with simulcast enabled and server side bandwidth probing disabled.

With server side bandwidth probing enabled other attendees will lose unrelated streams that never went offline as mentioned in https://github.com/aws/amazon-chime-sdk-js/issues/2559.

### Have you reviewed our existing documentation?

- [X] GitHub [issues](https://github.com/aws/amazon-chime-sdk-js/issues)
- [X] FAQs [page](https://aws.github.io/amazon-chime-sdk-js/modules/faqs.html)
- [X] README [page](https://github.com/aws/amazon-chime-sdk-js#amazon-chime-sdk-for-javascript)
- [X] Feature-specific [guides](https://github.com/aws/amazon-chime-sdk-js#guides)

### Reproduction steps

1) Join a meeting with multiple attendees
2) Turn on the camera for all of them
3) With one attendee disconnect the internet
4) Reconnect after 30 seconds
5) See the error mentioned above after a couple of seconds
6) Other attendees will see him with camera off

### Amazon Chime SDK for JavaScript version

3.10.0

### What browsers are you seeing the problem on?

Chromium

### Browser version

v109.0.5414.74

### Browser console logs

[attendee crashes on reconnect.log](https://github.com/aws/amazon-chime-sdk-js/files/10498146/attendee.crashes.on.reconnect.log)

**Comment by razvanduican12** [comment]
Hello,

@hensmi-amazon any updates?

Thanks!

**Comment by hensmi-amazon** [comment]
Unfortunately I ran into some unit test headaches there and it did not get https://github.com/aws/amazon-chime-sdk-js/pull/2565 merged in time for 3.11 . It is merged now and will make it into next release. Sorry for the delay.

**Comment by hensmi-amazon** [comment]
3.12 was released, sorry for delay.
